build.sbt

```diff
@@ -44,6 +44,7 @@ lazy val commonSettings = Seq(
     "org.apache.logging.log4j" % "log4j-api" % "2.23.1"
   ),
   resolvers += Resolver.mavenLocal,
+  resolvers += "Apache Spark 3.5.3 (RC2) Staging" at "https://repository.apache.org/content/repositories/orgapachespark-1466/",
```

Comment:
let's also test Delta 3.2.1 RC1: `resolvers += "Delta 3.2.1 (RC1) Staging" at https://oss.sonatype.org/content/repositories/iodelta-1166`

Reply:
there are some ignored tests in `TableReadWriteTest`, we can enable them now with this RC.

Reply:
sure